js/main.js

```diff
@@ -111,13 +123,37 @@ $newEntryPhotoUrl.addEventListener('input',function(e) {
 
 $journalEntry.addEventListener('submit', function(e) {
   e.preventDefault();
-  console.log('adasdasd')
   var title = $journalEntry.elements.newTitle.value;
   var image = $journalEntry.elements.newUrl.value;
   var notes = $journalEntry.elements.newNotes.value;
   var newEntryValues = {title,notes,image};
-  journal = newEntryValues;
+  journalAppend(newEntryValues);
+  journal.push(newEntryValues);
   $newEntryPhoto.src = './images/placeholder-image-square.jpg';
   $journalEntry.reset();
   dataView('entries');
 })
+
+function journalAppend(journal) {
+  var newLi = document.createElement('li');
+  newLi.setAttribute('class','journal-entry');
+  var div1 = document.createElement('div');
+  div1.setAttribute('class', 'col-half');
+  newLi.appendChild(div1);
+  var jImage = document.createElement('img');
+  jImage.setAttribute('src', journal.image);
+  jImage.setAttribute('class','full-width');
+  div1.appendChild(jImage);
+  var div2 = document.createElement('div');
+  div2.setAttribute('class','col-half flex-column');
+  newLi.appendChild(div2);
+  var heading = document.createElement('h2');
+  heading.setAttribute('class','profile-heading')
+  heading.textContent = journal.title;
+  div2.appendChild(heading);
+  var pBody = document.createElement('p');
+  pBody.textContent = journal.notes;
+  div2.appendChild(pBody);
+  var position = document.querySelector('#entriesUl');
+  position.prepend(newLi);
+}
```

Comment:
```
Define a function that takes a single journal entry object and returns a DOM tree that matches one of the example entries in the HTML.
```
This function takes a single journal entry object, converts it to a DOM tree and appends it to the dom.  The idea behind that particular instruciton is to write a pure function.  Your function's job is to take an object and return DOM tree.  You may then use the return value to append to the DOM.